app/src/main/java/com/example/plantsapp/data/RoomTasksRepository.kt

```diff
@@ -1,34 +1,42 @@
 package com.example.plantsapp.data
```

Comment:
Add new repository package in data , so all your repositories will be in this package